src/components/MultiHopTrade/components/TradeInput/components/TrendingTokens.tsx

```
@@ -1,39 +1,94 @@
+        <MenuOptionGroup type='radio' value={selectedCategory}>
+          {categoriesOptions.map(category => (
+            <MenuItemOption
+              value={category.value}
+              // eslint-disable-next-line react-memo/require-usememo
+              onClick={() => handleCategoryChange(category.value)}
+              fontSize='sm'
+              iconPlacement='end'
+              icon={checkedIcon}
+              color={selectedCategory === category.value ? 'text.primary' : 'text.subtle'}
+              fontWeight='bold'
+            >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)